machine.go

```diff
@@ -1067,3 +1067,88 @@ func (m *Machine) CreateSnapshot(ctx context.Context, memFilePath, snapshotPath
 	m.logger.Debug("snapshot created successfully")
 	return nil
 }
+
+// CreateBalloon creates a balloon device if one does not exist
+func (m *Machine) CreateBalloon(ctx context.Context, amountMb int64, deflateOnOom bool, statsPollingIntervals int64, opts ...PutBalloonOpt) error {
+	balloon := models.Balloon{
+		AmountMb:              &amountMb,
+		DeflateOnOom:          &deflateOnOom,
+		StatsPollingIntervals: statsPollingIntervals,
+	}
+	_, err := m.client.PutBalloon(ctx, &balloon, opts...)
+
+	if err == nil {
+		m.logger.Printf("Created balloon device with %v MB successful", amountMb)
```

Comment:
I think most of these m.logger.Printfs() should be m.Logger.Debugf(). What do you think?

Reply:
Yep, I just replaced all Printfs with Debugs()